media/components/sdfv/index.html

```diff
@@ -9,37 +9,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>SDFG Viewer</title>
 
-    <link rel="stylesheet" type="text/css" href="{{ NODE_MODULES }}/bootstrap/dist/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="{{ NODE_MODULES }}/jquery-editable-select/dist/jquery-editable-select.min.css"></script>
-    <link rel="stylesheet" type="text/css" href="{{ NODE_MODULES }}/material-icons/iconfont/material-icons.css">
-
-    <link rel="stylesheet" type="text/css" href="{{ NODE_MODULES }}/@spcl/sdfv/sdfv.css">
-
-    <link rel="stylesheet" type="text/css" href="{{ CSP_SRC }}/components/sdfv/sdfv_overrides.css">
-
     <script>
         // Reference to the VSCode API.
         let vscode = undefined;
-
-        let message_handler = undefined;
-
-        const csp_src = "{{ CSP_SRC }}";
     </script>
 
-    <script src="{{ NODE_MODULES }}/jquery/dist/jquery.min.js"></script>
-
     <script src="{{ NODE_MODULES }}/@spcl/sdfv/external_lib/pdfkit.standalone.js"></script>
```

Comment:
Unfortunately not yet, this is a soon (tm) change though. `node_modules` **needs** to be abolished entirely, because then we can **only** ship the `dist` folder (which is the goal). Unfortunately I need to put quite a bit more time into achieving that, since there are some issues currently.

Reply:
OK